client/signup/controller.js

```diff
@@ -144,6 +144,13 @@ export default {
 			}
 		}
 
+		// Record a `calypso_*` tracks event after clicking the launch button.
+		// We record the event here because the launch button itself is managed by the wpcom-block-editor app
+		// wpcom-block-editor app may be treated differently by the ad blocker but this code is guaranteed to be hit.
```

Comment:
I don't understand this bit — why would it be different? Anyway, we'll move the launch button to FSE plugin I think and we should be able to fire events from there too.